src/lib/docker/docker_image.js

```diff
@@ -18,7 +18,7 @@ const RETRY_CONFIG = {
   randomizationFactor: 0.25
 };
 
-export default class DockerImage {
+module.exports = class DockerImage {
```

Comment:
Maybe you could use babel plugin instead? https://www.npmjs.com/package/babel-plugin-add-module-exports